### DISCUSSION TITLE
Failed σε ανύπαρκτο test case στο 35-secretpaths

### DISCUSSION THREAD

**Discussion by panospask** [pdp-archive/pdp-archive.github.io#253] [General]
Καλημέρα και χρόνια πολλά.
Η υποβολή https://pdp-archive.com/pdp-judge/submission-status.php?id=5104 για το 35-secretpaths γίνεται fail στα testcases 47, 48, 49. Ωστόσο τα testcases του προβλήματος (https://drive.google.com/file/d/1HSFZLk8MGBYhzVr5T2Zftbafa-2lD6Ha/view) είναι 46.

**Comment by Dim131** [comment]
Καλημέρα Πάνο, 
Έχεις δίκιο, ευχαριστούμε! Τώρα πρέπει να διορθώθηκε, για έλεγξε

Αν βρεις κάποιο άλλο πρόβλημα ενημέρωσέ μας

**Reply by panospask** [reply]
Πραγματι, διορθώθηκε. Ευχαριστώ πολύ.
